src/docker-in-docker/install.sh

```diff
@@ -435,13 +435,14 @@ do
         eval "${dockerd_start}"
     fi
 
+    sleep 5s
```

Comment:
This will slow down all container starts by 5s, right?  I know we're trying to fix a problem that is likely racey, but waiting this long might be too big a performance hit?